Misc/NEWS

```diff
@@ -666,6 +666,9 @@ Library
 - Issue #24142: Reading a corrupt config file left configparser in an
   invalid state.  Original patch by Florian Höch.
 
+- bpo-29654 : Support If-Modified-Since HTTP header (browser cache). Patch
```

Comment:
Remove the space before colon. Use two spaces after sentence ending period. Move the entry to the start of the "Library" section (new entries are added at the start).